src/modules/actions/rename.jsx

```
@@ -40,8 +41,7 @@ export const rename = ({
     displayCondition: selection => {
       // special case for rename in sharings tab
       const isAllowedForSharedDrive = shouldHideIfSharedDriveRecipient
```

ESLint

[error] Replace `·shouldHideIfSharedDriveRecipient` with `⏎········shouldHideIfSharedDriveRecipient·&&`

(prettier/prettier)